packages/javascript-sdk/src/oauth2-client/interfaces.ts

```diff
@@ -35,7 +36,7 @@ interface AccessTokenResponse {
 /**
  * Options used when requesting the authorization URL.
  */
-interface GetAuthorizationUrlOptions extends ConfigOptions {
+interface GetAuthorizationUrlOptions extends GetTokensOptions {
```

Comment:
Was this intentional?

Reply:
This doesn't feel right to me as the hierarchy of this is in reverse. The `TokenManager` class, on which `getTokens` is found, internally uses the `OAuthClient` class. So, if one inherited from the other, I would think `GetTokensOptions` would extend `GetAuthorizationUrlOptions`, not the other way around, yeah? But, ultimately, I don't think either extend the other. For example, `forceRenew` and `login` are not properties of `GetAuthorizationUrlOptions` as the `createAuthorizeUrl` doesn't use them.

Reply:
we resolved this on slack. its right and wrong and i've created a ticket to complete a fix for this.